add shadowlands mount data on release

slemke opened the pull request:
* added missing kyrian mounts from the api
* moved some mounts to different categories, because the source is now known/corrected/changed

seirl commented on app/data/mounts.json:
```diff
@@ -225,6 +225,13 @@
+          },
+          {
+            "ID": 1414,
+            "name": "Sinrunner Blanchy",
```
Why in treasures? This is a rare spawn according to wowhead.

slemke replied:
Is now a rare spawn.

seirl commented on app/data/mounts.json:
```diff
@@ -291,6 +298,13 @@
+          },
+          {
+            "ID": 1382,
+            "name": "Inquisition Intimidator",
```
This should go in the Venthyr covenant, as it's more restrictive than "reputation". All people in the Venthyr covenant can buy it if they get exalted, but not every exalted people can get it since they need to be in the correct covenant.

slemke replied:
Moved to Venthyr.

seirl commented on app/data/mounts.json:
```diff
@@ -322,6 +336,19 @@
+          },
+          {
+            "ID": 1437,
+            "name": "Battle-Bound Warhound",
```
This doesn't drop from a dungeon, this drops from the non-instanced bosses of Theater of Pain.

slemke replied:
Is now grouped with the maw mount under "Zone". Not a zone wide drop, but both drop in a zone. I couldn't come up with something better, but I didn't want these mounts to have their own category.

seirl commented on app/data/mounts.json:
```diff
@@ -632,6 +734,20 @@
+          },
+          {
+            "ID": 1367,
+            "name": "Chosen Tauralus",
```
Achievement rewards should go in the achievement section for consistency reasons.

slemke replied:
I think we need some sort of rule on where to put stuff with shadowlands mounts. This mount does come from an achievement where you need a specific covenant (necrolords) and is part of the covenant sanctum. If we put this under achievements we also need to move other mounts. For example, some mounts from rare mobs drop only for specific covenants.

I think we can do it two ways:

1. Keep the old order, there are no covenant sections. Every mount goes into quest, achievement, drop, whatever.
2. Move every mount that has a covenant requirement under its specific covenant where it does not matter if it is a drop, achievement, ...

Based on that I would redo this PR since a lot more stuff is uncovered now and the old data is not even valid anymore.

seirl replied:
Yeah, I think we should go with 2. I hadn't realized this mount was covenant-specific.

slemke replied:
Covenant specific mounts are now under their covenants. Mounts that drop from rare spawns that only require covenant interaction are still under rare spawn, because they still can be looted by everyone.

seirl commented on app/data/mounts.json:
```diff
@@ -632,6 +734,20 @@
+          },
+          {
+            "ID": 1371,
+            "name": "Armored Chosen Tauralus",
```
Ditto

seirl reviewed (changes requested):
Thanks. A lot of mounts don't seem to be in the correct section, please take a look at the comments below.

slemke commented:
I moved a few mounts, because of updated drop information. I grouped the theater of pain and the maw mount under "Zone" like the mounts in BFA. They are not a zone wide drop, but I couldn't come up with something better. The covenant specific mounts are now under their respective covenant section. Some of the rare spawns require covenant specific interaction, but can be looted without joining the covenant that is required for the interaction. Example: rare mob requires night fae covenant member to disable the shield so the mob is attackable, but a necrolord player can loot the mount once the rare spawn is dead. So I left them under "Rare Spawn". I also moved the oozing necroray eggs to make the rows look better on the page.

seirl commented:
Looks great, thanks!
